pages/annotation.vue

```diff
@@ -65,6 +65,11 @@
             };
         },
 
+        mounted() {
```

Comment:
is this a leftover thing? Can we remove it?

Reply:
I think so @jarmoza can you confirm?

Reply:
@rmanaem Correct. This can be removed. This was just for debugging the state of the `columnToCategoryMap` on annotation page mount.